sr_data_visualization/CMakeLists.txt

```diff
@@ -0,0 +1,39 @@
+cmake_minimum_required(VERSION 2.8.3)
+project(sr_data_visualization)
+
+find_package(catkin REQUIRED COMPONENTS
+  rospy
+  rqt_gui
+  rqt_gui_py
+  std_msgs
+  sr_robot_msgs
+)
+
+catkin_python_setup()
+
+catkin_package(
+  CATKIN_DEPENDS
+    rospy
+    rqt_gui
+    rqt_gui_py
+    std_msgs
+    sr_robot_msgs
+  INCLUDE_DIRS include
+)
+
+include_directories(include ${Boost_INCLUDE_DIRS} ${catkin_INCLUDE_DIRS})
+
+#add_executable(sr_moveit_run_benchmark src/SrRunBenchmark.cpp src/SrBenchmarkExecutor.cpp)
+#target_link_libraries(sr_moveit_run_benchmark ${catkin_LIBRARIES})
+
+install(DIRECTORY launch DESTINATION ${CATKIN_PACKAGE_SHARE_DESTINATION})
+install( FILES sr_data_visualizer_plugin.xml DESTINATION ${CATKIN_PACKAGE_SHARE_DESTINATION} )
+## Add folders to be run by python nosetests
+# catkin_add_nosetests(test)
+# if (CATKIN_ENABLE_TESTING)
+#     find_package(rostest REQUIRED)
+#     add_rostest(test/test_import_export_scenes.test)
+#     add_rostest(test/test_import_export_queries.test)
+# endif ()
+
+
```

Comment:
Unless I'm missing something, commented stuff is from another package?

Reply:
Oops, yeah, thanks :)